src/test/resources/features/refapp-2.x/12-condition/conditions.feature

```diff
@@ -11,7 +11,7 @@ Feature: Patient's Conditions Managment
     Then System on Add New Condition Page
     And User enters "<activity>" condition
     And User clicks save
-    Then System loads Manage Conditions Page
+    Then System adds New Condition in Conditions table
```

Comment:
Here are things that you may need to know;

1. Each scenario is independent from the other and when its run is done, the test patient is deleted from the db and when the test runs the next scenario it first creates its own test patient.
2. Scenarios that are failing is as a result of their steps missing adding condition in the condition's table so that the preceding steps can pass. Eg
```js
  @selenium
  @condition
  Scenario: Set first condition to inactive
    And User clicks on set inactive button
    Then System should move condition to inactive section
``
a use can can not click on set inactive button unless there a condition in the conditions table.

Reply:
> @mherman22 here are things that you may need to know;
> 
> 1. Each scenario is independent from the other and when its run is done, the test patient is deleted from the db and when the test runs the next scenario it first creates its own test patient.
> 2. Scenarios that are failing is as a result of their steps missing adding condition in the condition's table so that the preceding steps can pass. Eg
> 
> ```js
>   @selenium
>   @condition
>   Scenario: Set first condition to inactive
>     And User clicks on set inactive button
>     Then System should move condition to inactive section
> ```
> 
> set inactive button can only be visible on the Conditions page when there a condition in the conditions table.

thanks @kdaud for that clarity, i thought when i added `Then System adds New Condition in Conditions table` in the Scenario Outline i was catering for that.

Reply:
@kdaud  I think this might fail since [Qa-server is down](https://qa-refapp.openmrs.org/openmrs/)

Reply:
Maybe Re-run if it is up and running

Reply:
> I think this might fail since [Qa-server is down](https://qa-refapp.openmrs.org/openmrs/)

is it still down?, cuz it's running in my browser

Reply:
> Maybe Re-run if it is up and running

to add to this, am not quite sure why its failing against the server but running successfully locally on my pc

Reply:
It up and Running @mherman22 but am wondering Github actions seem not to run now on this PR ....